drivers/SmartThings/zigbee-contact/src/smartsense-multi/init.lua

```diff
@@ -146,7 +146,19 @@ local function xyz_handler(driver, device, zb_rx)
   local x = multi_utils.convert_to_signedInt16(zb_rx.body.zcl_body.body_bytes:byte(1), zb_rx.body.zcl_body.body_bytes:byte(2))
   local y = multi_utils.convert_to_signedInt16(zb_rx.body.zcl_body.body_bytes:byte(3), zb_rx.body.zcl_body.body_bytes:byte(4))
   local z = multi_utils.convert_to_signedInt16(zb_rx.body.zcl_body.body_bytes:byte(5), zb_rx.body.zcl_body.body_bytes:byte(6))
-  multi_utils.handle_three_axis_report(device, x, y, z)
+  device:emit_event(capabilities.threeAxis.threeAxis({value = {x, y, z}}))
+  if device.preferences["certifiedpreferences.garageSensor"] then
+    -- The sensor is mounted on the garage door vertically. Unlike the newer sensors, the z-axis is parallel to the ground
+    -- when the door is closed and perpendicular to the ground when the door is open. This is why we are using custom handling
+    -- instead of multi_utils.handle_three_axis_report. The values here were the same used in the original Groovy DTH
+    -- and in the protocol handler.
+    local abs_z = math.abs(z)
+    if abs_z > 825 then
```

Comment:
What if you just added an argument to the `multi_utils` function to use as the value here. It's a lot of code repetition to handle changing `900` to `825`.

Reply:
In my opinion that would make the function harder to read than just copying the few lines we need right here. The change that is more important than the 900 -> 825 is treating the larger values as open instead of closed.

Reply:
ah i didn't notice the flip there